valhalla/sif/edgelabel.h

```diff
@@ -443,8 +458,14 @@ class EdgeLabel {
   // Flag indicating edge is an unpaved road.
   uint32_t unpaved_ : 1;
   uint32_t has_measured_speed_ : 1;
+
   // Flag if this edge had HGV access
   uint32_t hgv_access_ : 1;
+  // Flag indicating edge is a bridge.
+  uint32_t bridge_ : 1;
+  // Flag indicating edge is a tunnel.
+  uint32_t tunnel_ : 1;
+
   uint32_t spare : 12;
```

Comment:
We added two bits for bridge and tunnel label, shouldn't the spare then be subtracted by 2?
So it would be `uint32_t spare : 10`

Reply:
Oh yes, good catch!

Reply:
Updated it